std_msgs/Header header
common/Uint8TimeStamped adcu_cld_headlampcmd
common/Uint8TimeStamped adcu_cld_dblflashlampcmd
common/Uint8TimeStamped adcu_cld_turnllampcmd
common/Uint8TimeStamped adcu_cld_turnrlampcmd
common/Uint8TimeStamped adcu_cld_backlampcmd
common/Uint8TimeStamped adcu_cld_buzzercmd
common/Uint8TimeStamped adcu_cld_horncmd
common/Uint8TimeStamped adcu_cld_runlampcmd